src/utils/isShiftEnter.ts

```diff
@@ -0,0 +1,18 @@
+//Control the Shift+enter command
+const isShiftEnter = (function () {
```

Comment:
This utility section looks weird next to the others (file, editor, statistics...), I'd prefer if we could do it without it, if we absolutely needed it I would be more comfortable putting this function into an "autoSymbol" or "asi" (Automatic Symbol Insertion? 🤔) utils section or something.

The API of this function is weird, we are kind of calling functions via strings, instead there should be a "get" and "set" functions provided.

Anyway if I'm understanding this correctly when "shift+enter" is pressed we set this flag (actually we toggle it, which may be a bug), then we listen for changes in `changes.ts` and we check if the newline inserted was actually inserted by pressing shift (if the flag is set), in that case we do nothing, is this correct? 

Couldn't/shouldn't we instead listen for the "enter" and "shift+enter" shortcuts instead of listening for changes? Like, we actually care about those shortcuts, not if somebody pastes a new line or deletes a new line.